puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-puppet-pc1-systest.yaml

```diff
@@ -0,0 +1,31 @@
+- job:
+    name: foreman-puppet-pc1-systest
+    node: 'el && ipv6'
+    project-type: matrix
+    logrotate:
+      daysToKeep: 8
+    scm:
+      - forklift
+    triggers:
+      - timed: 'H 10 * * 6'
+    axes:
+      - axis:
+          type: user-defined
+          name: os
+          values:
+            - centos7
+    builders:
+      - systest:
+          repo: nightly
+          repo_environment: release
+          pl_puppet: 4
```

Comment:
@ekohl Are we all Puppet 5 at this point? Should we update this to 5 or drop this job all together?

Reply:
I think we still support Puppet 4 for now. We could make an official statement for 1.18 that we're going to stop testing it which would make it unsupported.